packages/libs/chainweb-stream-client/src/index.ts

```diff
@@ -273,8 +372,12 @@ class ChainwebStream extends EventEmitter {
     if (this.limit !== undefined) {
       urlParamArgs.push(['limit', String(this.limit)]);
     }
+    // TODO This reconnection strategy of -3 from last max height
+    // guarrantees that we will not miss events, but it also means
+    // that confirmed transactions will be emitted more than once
+    // Discussion here: https://github.com/kadena-community/kadena.js/issues/275
     if (this._lastHeight !== undefined) {
-      urlParamArgs.push(['minHeight', String(this._lastHeight)]);
+      urlParamArgs.push(['minHeight', String(this._lastHeight - 3)]);
```

Comment:
This also fixes a missed events edge case case when reconnectin.

Request for comments regarding what the behaviour should be regarding duplicates https://github.com/kadena-community/kadena.js/issues/275